CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing
+
+When contributing to this repository, please first propose the changes you wish to make in an issue first. Doing this gives us the chance to discuss your proposal, add some feedback and provide assistence where needed.
+
+Please note we have a [code of conduct](./CODE_OF_CONDUCT.md). We expect every contributor to follow the same standards we do.
+
+## Submitting a pull request
+
+1. Fork the repository and clone to your development environment
+2. Create a new branch: `git checkout -b my-branch-name`
+3. Implement your changes
+5. Push to your fork and submit a pull request
```

Comment:
All the project will require the contributors to sign either a CLA or DCO before any contributions can be accepted. Please add a point about that as well.

Reply:
I went with the local repository CONTRIBUTING file as it has some specifics after all, and we are not going to have CLA nor DCO requirement.